src/Control/Email/Email.php

```diff
@@ -933,23 +967,29 @@ public function findPlainPart()
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::getTextBody()
+     *
      * @return bool
      */
     public function hasPlainPart()
     {
+        Deprecation::notice('4.12.0', 'Will be replaced with Symfony\Component\Mime\Email::getTextBody()');
```

Comment:
```suggestion
        Deprecation::notice('4.12.0', 'Use getTextBody() instead');
```

Reply:
Should say replace with not equivalent as discussed above.

Reply:
Updated